testsuite/cluster-test/src/main.rs

```diff
@@ -37,11 +37,17 @@ const HEALTH_POLL_INTERVAL: Duration = Duration::from_secs(5);
 #[derive(StructOpt, Debug)]
 #[structopt(group = ArgGroup::with_name("action").required(true))]
 struct Args {
-    #[structopt(short = "w", long)]
+    #[structopt(short = "w", long, default_value = "")]
```

Comment:
Could be better to make workplace to be `Option<String>` and have None as default
Also can add `conflicts_with="swarm"`

Reply:
Updated.